packages/react/src/components/Footer/LocaleButton.js

```diff
@@ -19,11 +20,34 @@ const { prefix } = settings;
  * EXPERIMENTAL: Renders the locale button
  *
  * @private
+ *
+ * @param {Function} selectItem method to handle selected item
+ *
  * @returns {object} JSX object
  */
-const LocaleButton = () => {
+const LocaleButton = ({ selectItem }) => {
   const [isOpen, setIsOpen] = useState(false);
 
+  // retrieve list from local storage
+  const list = JSON.parse(localStorage.getItem('countryList'));
```

Comment:
I know this is still just experimental, but we should probably get this data from the service rather than fetching right from localStorage. Let the service determine if it's a new net call or fetch from localStorage.

Reply:
The service gets called in the `TranslationAPI` service and does the work of checking sessionStorage/making a new call and setting session storage

Reply:
Right, but the getter should be also from the service and not in here. Essentially, make a call to `LocaleAPI.getList`, and it should return it. 

In the API, it should check if the localStorage is already set. If it is, return that. If not, make the axios call, set it, then return the list.

Reply:
done, the locale is retrieved by calling `LocaleAPI.getLocale` which will determine the locale from either the `lang` attr or the cookie. I'm keeping the cookie for instances where the `lang` isn't there.